tests/pubsub/testgossipsub.nim

```diff
@@ -807,8 +815,6 @@ suite "GossipSub":
 
     await allFuturesThrowing(nodes.mapIt(allFutures(it.switch.stop())))
```

Comment:
it would be nice to refactor these line, across all tests 
`await allFuturesThrowing(nodes.mapIt(allFutures(it.switch.stop())))`
to start(), defer stop() similarly how  `testgossipsubparameters` file use it...